arboretum/common/kube_utils.py

```diff
@@ -0,0 +1,44 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common Kubernetes functions."""
+
+from requests import HTTPError
+
+
+def get_cluster_resources(session, token, resource_types, verify=False):
```

Comment:
I _think_ I understand why we want to have this `verify=False` but it really seems something that we need to justify very well or try to avoid as much as we can. Is it possible to get a trusted connection to the cluster some how? I would like to avoid the idea of having `verify=False`.

Reply:
sure, I've removed this.